### DISCUSSION TITLE
jaeger exporter

### DISCUSSION THREAD

**Discussion by itsmaneesh** [open-telemetry/opentelemetry-java-instrumentation#5754] [Q&A]
Hi,
I am using Jaeger as exporter. But, Can you please clarify whether the OTEL_EXPORTER_JAEGER_ENDPOINT env variable shall have the value of Jaeger Agent or Jaeger Collector?  Or It can be either of them with appropriate port numbers?

It would be great if we you can provide example values to have more clarity.

Thank you.

**Comment by trask** [comment]
hi @itsmaneesh, let us know if this helps: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#jaeger-exporter
